reverse_script.py

```diff
@@ -0,0 +1,64 @@
+import os, sys
+import re
+import json
+
+# DESCRIPTION

```

Comment:
Use docstrings for documentation
https://devman.org/qna/13/chto-takoe-docstring-s-chem-ego-edjat/

Reply:
removed in d608ea5dd37b093be0818a47c482cc653772dd49